class/person.rb

```diff
@@ -29,4 +30,8 @@ def of_age?
   def generate_id
     rand(1..1000)
   end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] Great job so far creating the associations between your classes. However, In your method for adding rentals please make sure to pass the correct `parameters`. The person itself should be sent as a `parameter` to create the new rental achieving the `has-many` association.  Instead of passing the rental, you should pass `(book, date)`. Your code should look like below.

```
 def add_rental(date, book,)
   Rental.new(date, book, self)
  end
```